CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing
+
+## General Workflow
+
+1. Fork the repo
+1. Cut a namespaced feature branch from master
+1. Make commits to your feature branch. Prefix each commit like so:
+1. When you've finished with your fix or feature, merge upstream changes into your branch. submit a pull request directly to master. Include a description of your changes.
```

Comment:
PRs should be used whenever a conversation wants to be started.
pulling from this guy's [take on github flow](http://scottchacon.com/2011/08/31/github-flow.html) 
The PR-maker can just @ someone if they want their attention.
the [WIP] pre/postfix lets us know it's not for merging.

I guess the question would be if PRs should be closed once the question is answered and reopened when a new one comes up or if a new PR should be made.

thoughts on all this?